frontend/src/test/java/com/google/tripmeout/serialization/GsonPlaceVisitTypeAdapterTest.java

```diff
@@ -55,8 +55,14 @@ public void deserialize_noPlaceId_throwsJsonParseException() throws Exception {
   }
   @Test
   public void deserialize_noName_throwsJsonParseException() throws Exception {
```

Comment:
Missed this on my first pass. Please rename the method to reflect the new behavior. Looks like you'll need to rename all changed methods.